x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ml/action/InternalInferModelActionRequestTests.java

```diff
@@ -27,7 +24,7 @@
 import java.util.stream.Stream;
 
 
-public class InternalInferModelActionRequestTests extends AbstractBWCWireSerializationTestCase<Request> {
+public class InternalInferModelActionRequestTests extends AbstractWireSerializingTestCase<Request> {
```

Comment:
I think we should keep this as `AbstractBWCWireSerializationTestCase` But the `mutateInstanceForVersion` clauses could be removed.

This way we catch if anything funky changes in serialization between 7.x and 8